debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -253,6 +253,14 @@ protected void processRow(LogMinerEventRow row) throws SQLException, Interrupted
         if (!row.getEventType().equals(EventType.MISSING_SCN)) {
             lastProcessedScn = row.getScn();
         }
+        // filter out all events that are captured as part of the initial snapshot
+        if (row.getScn().compareTo(offsetContext.getSnapshotScn()) < 0) {
+            Map<String, Scn> snapshotPendingTransactions = offsetContext.getSnapshotPendingTransactions();
+            if (snapshotPendingTransactions == null || !snapshotPendingTransactions.containsKey(row.getTransactionId())) {
+                LOGGER.info("Skipping event {} (SCN {}) because it is already encompassed by the initial snapshot", row.getEventType(), row.getScn());
```

Comment:
Let's reduce this verbosity to at least `DEBUG` level.  I'm afraid this could be overly verbose in some situations.